resource/ui/ts/models/timeseries.ts

```diff
@@ -7,6 +7,8 @@
  */
```

Comment:
Can you fix the source comment at the top of the file?
Also, perhaps this should be named Metrics instead of timeseries?